portal/package.json

```
@@ -24,7 +24,7 @@
     "lint": "eslint ./server ./scripts",
     "lint:fix": "eslint ./server ./scripts --fix",
     "start": "node server/index.js",
-    "test": "jest --coverage --verbose --config=jest.config.js",
+    "test": "jest --verbose --config=jest.config.js",
```

```suggestion
    "test": "jest --coverage --verbose --config=jest.config.js",
```
this was removed accidentally

---

Thanks, please revert the removal.